src/main/resources/db/changelog/changes/02-create-users-table.yaml

```diff
@@ -25,7 +25,7 @@ databaseChangeLog:
                   constraints:
                     nullable: false
               - column:
-                  name: lastName
+                  name: last_name
```

Comment:
Usually we don't edit already existing migration files. For any new change a new migration script must be created.

Reply:
Yeah, I know. But I have faced some weird behavior. Scrip 02 was executed with a wrong column name. Then I created 03 script to fix the name. It worked well. But then, when I added new scripts the 03 started to throw an error that old column doesn't exist. I've spent a lot of time trying to fix it, but then I just decided to rollback scripts 02 and 03